common/meta/types/proto/meta.proto

```diff
@@ -87,6 +83,71 @@ message WatchResponse {
   Event event = 1;
 }
 
+// messages for txn
+message TxnCondition {
+  // compare target enum
+  enum ConditionTarget {
+    Value = 0;
+    Seq = 1;
+  }
+
+  // condition result
+  enum ConditionResult {
+    Equal = 0;
+    Greater = 1;
+    Less = 2;
+    NotEqual = 3; 
+  }
+
+  string key = 1;
+  ConditionTarget target = 2;
+
+  oneof target_union {
+    // used when compare value
+    bytes value = 3;
+    // used when compare seq
+    uint64 seq = 4;
+  }
```

Comment:
There is already `enum ConditionTarget`. It looks `enum ConditionTarget` can be removed?

Reply:
remove `ConditionTarget` enum.

Reply:
done